hw/ip_templates/ac_range_check/rtl/ac_range_check.sv.tpl

```diff
@@ -187,6 +187,7 @@ module ${module_instance_name}
   // Priority comparison. If the deny mask is larger than the read, write, or execute mask, there
   // was an address match with a higher priority for the range to be denied
   logic read_allowed, write_allowed, execute_allowed;
+
```

Comment:
Accidental newline? :-)

On the commit message that contains this change: Isn't it the CTN _SRAM_ rather than the CTN _flash_?

Reply:
The newline is lingering from changes that were merged separately. Any reference to CTN flash is a recall failure on my part; for some time I thought the device was supposed to be a flash memory.